triss-core/src/main/kotlin/com/pp/trisscore/service/ComparisonService.kt

```diff
@@ -12,8 +12,19 @@ import org.springframework.stereotype.Service
 @Service
 class ComparisonService {
 
-    fun compareApplicationsInfo(dbApplicationInfo: ApplicationInfo, reqApplicationInfo: ApplicationInfo, role: Role) {
-        compareApplications(dbApplicationInfo.application, reqApplicationInfo.application, role)
+    fun compareApproveApplicationsInfo(dbApplicationInfo: ApplicationInfo, reqApplicationInfo: ApplicationInfo, role: Role) {
+        compareApplications(dbApplicationInfo.application, reqApplicationInfo.application,role)
+        compareApproveApplication(role, dbApplicationInfo.application, reqApplicationInfo.application)
+        compareInstitutes(dbApplicationInfo.institute, reqApplicationInfo.institute)
+        if (role != Role.DIRECTOR)
```

Comment:
could you add braces in every if? It's easier to read when we have it everywhere